============================= test session starts =============================
platform win32 -- Python 3.13.2, pytest-8.3.5, pluggy-1.5.0
rootdir: c:\Users\Dharmi\OneDrive\Desktop\Assignment 3
plugins: anyio-4.9.0, cov-6.1.0, flask-1.3.0
collected 2 items

test_.py FF                                                              [100%]

============================== warnings summary ===============================
test_.py::test_score
  C:\Users\Dharmi\AppData\Local\Programs\Python\Python313\Lib\site-packages\sklearn\utils\validation.py:2739: UserWarning: X does not have valid feature names, but SVC was fitted with feature names
    warnings.warn(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=============================== tests coverage ================================
_______________ coverage: platform win32, python 3.13.2-final-0 _______________

Name       Stmts   Miss  Cover
------------------------------
score.py      30      2    93%
test.py       43      0   100%
-------------------------------
TOTAL         73      2    97%
Coverage HTML written to dir coverage.txt

================================= 2 passed, 19 warnings in 10.26s ==================================
